### DISCUSSION TITLE
[Proposition] Ajouter l'action personnalisée Coverage pour calculer le coverage

### DISCUSSION THREAD

**Discussion by Mozinou2013** [INFOM126-Automated-Software-Engineering/2048-groupe-07#32] [New GitHub action]
Nom de l'action proposée : coverage
Objectif de l'action (décrire ce qu'elle fait et pourquoi elle est nécessaire) : Calculer le coverage
Fonctionnement (expliquer brièvement les étapes et les technologies utilisées) : Calculer le coverage à l'aide de JaCoCo

**Comment by ldelpier** [comment]
Cela m'a l'air intéressant mais quels sont les fichiers ?

**Reply by Mozinou2013** [reply]
Les fichiers ajoutés seraient un fichier yml et une documentation si nécessaire

De plus, pour faire du Coverage avec Java, il faut ajouter JaCoCO dans le projet. Du coup, le fichier pom.xml changerait et on y ajouterait la configuration de JaCoCo

**Reply by ldelpier** [reply]
Super
